examples/nodejs/package.json

```diff
@@ -6,8 +6,8 @@
   "repository": "https://github.com/dagger/dagger",
   "author": "Jean-François Farge",
   "license": "MIT",
-  "dependencies": {
-    "@dagger.io/nodejs-sdk": "https://github.com/slumbering/dagger-nodejs-sdk.git",
+  "devDependencies": {
+    "@dagger.io/dagger": "^0.1.0-alpha.4",
```

Comment:
alpha packages are going to be removed. 
`"@dagger.io/dagger": "^0.1.0",` should be the published package version.
thanks